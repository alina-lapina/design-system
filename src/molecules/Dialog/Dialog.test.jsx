// import React from 'react';
// import { shallow } from 'enzyme';
// import Dialog from './Dialog';
//
// describe('Dialog molecule', () => {
// 	test('Matches the snapshot', () => {
// 		const dialog = shallow(<Dialog text="Some text" title="A title" />);
// 		expect(dialog).toMatchSnapshot();
// 	});
// });