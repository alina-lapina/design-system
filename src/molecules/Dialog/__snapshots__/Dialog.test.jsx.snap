// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog molecule Matches the snapshot 1`] = `
<div
  className="dialog-container info"
>
  <div
    className="icon-bar"
  >
    <i
      data-feather="alert-circle"
    />
  </div>
  <div
    className="info-content"
  >
    <p>
      <strong>
        A title
      </strong>
    </p>
    <p>
      <span
        className="dialog-text"
      >
        Some text
      </span>
    </p>
  </div>
</div>
`;
